src/bvals/comms/boundary_communication.cpp

```diff
@@ -310,38 +300,28 @@ TaskStatus ApplyCoarseBoundaryConditions(std::shared_ptr<MeshData<Real>> &md) {
   return stat;
 }
 
+TaskStatus ApplyFineBoundaryConditions(std::shared_ptr<MeshData<Real>> &md) {
+  if (!md->GetMeshPointer()->multilevel) return TaskStatus::complete;
+  TaskStatus stat = TaskStatus::complete;
+  for (int block = 0; block < md->NumBlocks(); ++block) {
+    auto bstat = ApplyBoundaryConditionsOnCoarseOrFine(md->GetBlockData(block), false);
+    // if (bstat != TaskStatus::complete) stat = bstat;
```

Comment:
Why is this commented out?

Reply:
Oh, I should probably just remove that. `ApplyBoundaryConditionsOnCoarseOrFine` always returns `TaskStatus::complete`.

Reply:
done